projects/js-packages/components/components/button/stories/Button.mdx

```diff
@@ -35,21 +35,21 @@ It accepts 2 size types: `normal` and `small`.
 
 ### weight
 
-It accepts 2 size types: `bold` and `regular`. `bold` as default.
+It accepts 2 weight types: `bold` and `regular`. `bold` as default.
 
 ### icon
 
-It accepts an SVG, Component or string.
+It accepts a SVG, Component or string.
```

Comment:
The correct is actually "an" - https://editorsmanual.com/articles/indefinite-article-a-an-with-abbreviations/

Reply:
> a NATO member but an NBA player
Even though both abbreviations begin with “N,” NATO (pronounced “NAY-toh”) begins with a consonant sound, while NBA (pronounced “en-bee-AY”) begins with a vowel sound.

TIL :exploding_head: